```
try:
    num = 10 / 0
except:
    print("An error happened...")
```

`except:` can be followed by the name of an error for more precise handling.